deploy/base/operator.yaml

```diff
@@ -18,6 +18,25 @@ spec:
     spec:
       serviceAccountName: seccomp-operator
       initContainers:
+        - name: extract-seccomp-operator-profile
+          # TODO: replace this with golang app
+          image: bitnami/kubectl:1.19.2
+          command:
+          - bash
+          - -c
+          args:
+          - |
+            set -euo pipefail
+
+            while ! kubectl get crd | grep seccompprofile ; do sleep 1 ; done
+
+            kubectl apply -f /opt/bootstrap-profiles/
+            kubectl --namespace seccomp-operator get seccompprofile seccomp-operator -o jsonpath='{.spec}' > /opt/seccomp-profiles/seccomp-operator.json
+          volumeMounts:
+          - name: profile-configmap-volume
+            mountPath: /opt/bootstrap-profiles
+          - name: seccomp-operator-profile-volume
+            mountPath: /opt/seccomp-profiles
```

Comment:
I think we can do it this way. 

An alternative idea would be:

- Give the operator permissions to write CRDs (we need it anyway later on with the recording feature)
- Deploy the profile on operator startup, containing the same logic for checking if the CRD is deployed (via client-go)
- Let the operator reconcile…

I know that I've mentioned it somewhere in the past, do you think this would work?

Reply:
But the operator can't even start if it requires the seccomp-operator profile which doesn't exist yet, that's the chicken-and-egg problem I was describing

https://github.com/kubernetes-sigs/security-profiles-operator/blob/fb21e8a423c7fbc824754b73ed625d65e382e7f9/deploy/operator.yaml#L79

Reply:
Yes, makes sense. Can we merge both init containers into one?

Reply:
Hmm yes I think they could be merged. But it would mean using the same `securityContext` for both parts, where currently this init container doesn't need any `securityContext`, is that okay?

Reply:
Yes we can extend the existing security context to match the new requirements.